scripts/bdd-tests.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Generate the OpenApi client code for the BDD tests.
+# The autogenerated code is placed in ../tests/bdd/openapi
+./scripts/generate-openapi-bindings-bdd.sh
+
+# Remove the BDD tests for the autogenerated code (we aren't interested in those).
+rm -rf tests/bdd/openapi/test
```

Comment:
these path's should all be relative to the ROOT_DIR, eg:
```bash
#!/usr/bin/env bash

set -euxo pipefail 

SCRIPTDIR="$(dirname "$0")"
export ROOT_DIR="$SCRIPTDIR"/..

# Generate the OpenApi client code for the BDD tests.
# The autogenerated code is placed in ../tests/bdd/openapi
"$SCRIPTDIR"/generate-openapi-bindings-bdd.sh

# Remove the BDD tests for the autogenerated code (we aren't interested in those).
rm -rf "$ROOT_DIR"/tests/bdd/openapi/test

virtualenv --no-setuptools "$ROOT_DIR"/tests/bdd/venv
# shellcheck disable=SC1091
source "$ROOT_DIR"/tests/bdd/venv/bin/activate
pip install -r "$ROOT_DIR"/tests/bdd/requirements.txt

# Configure the environment variables used by the pytests
export PYTHONPATH=$PYTHONPATH:"$ROOT_DIR"/tests/bdd/openapi

pytest "$ROOT_DIR"/tests/bdd
```

Reply:
Thanks. Done.